src/zjs_console.c

```diff
@@ -253,33 +253,36 @@ static jerry_value_t console_assert(const jerry_value_t function_obj,
                                     const jerry_length_t argc)
 {
     char message[MAX_STR_LENGTH];
-    uint8_t has_message = 0;
     if (!jerry_value_is_boolean(argv[0])) {
         ERR_PRINT("invalid parameters\n");
         return ZJS_UNDEFINED;
     }
-    bool b = jerry_get_boolean_value(argv[0]);
-    if (!b) {
+    if (argv[1]) {
```

Comment:
You can't do this, you don't even know if there *is* a second argument yet. It's an optional argument, and even if it wasn't you still have to make sure it's there.